docs/Using-Fleet/Audit-Activities.md

```diff
@@ -318,6 +318,27 @@ This activity contains a field "teams" where each item contains the team details
 }
 ```
 
+### Type `transferred_hosts`
+
+Generated when a user transfers a host (or multiple hosts) to a team (or no team).
+
+This activity contains the following fields:
+- "team_id": The ID of the team that the hosts were transferred to, null if transferred to no team.
```

Comment:
Would suggest a tiny formatting change:
```suggestion
- "team_id": The ID of the team that the hosts were transferred to,`null` if transferred to no team.
```

Reply:
@rachaelshaw The change makes sense to me but there are a number of places where "null" is used already in activities' description (e.g. https://fleetdm.com/docs/using-fleet/audit-activities#type-code-edited-agent-options-code), it should be consistent - would you prefer to change them all to use the backticks? (it's not that big a change, just to double-check given how the others are already documented)

Reply:
@mna good point, I should have looked at the rest of the page for context— best to be consistent. (I think we _should_ change that formatting so they all have backticks, but since it's a lot of little changes that aren't related to the issue for this PR, that totally doesn't need to happen now. I'm keeping track of formatting-related stuff like this in a doc for now, so will add a note there.)

Reply:
@rachaelshaw thanks, FYI I did end up making the change in that PR.